README.md

```diff
@@ -244,6 +245,15 @@ if (_.contains(projectsInRGSoC, {
 
 Now, we're officially a part of [leapcode.io](http://leapcode.io/). They're still building and hopeful to have their platform up pretty soon. [Click here](https://www.leapcode.io/) to get an early access.
 
+## Hacktoberfest
+
+<a href="https://hacktoberfest.digitalocean.com/"><img src="assets/Logo.svg" width="30%"></a>
```

Comment:
Can we get `.png` / `.jpg` logo?

Reply:
There is no .png/ .jpg available.
Only svg logo is available in the assets of hacktoberfest.

Reply:
@nandikajain Got it. Merging it.